apps/edxapp/templates/services/cms/_dc_base.yml.j2

```diff
@@ -113,8 +113,6 @@ spec:
           name: edxapp-config
```

Comment:
Concerning your commit title, there is an emoji for reverting things in gitmoji: :rewind: